tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -477,25 +477,32 @@ module.exports = {
      * @returns {Array} array of sharing indicator
      */
     getShareIndicatorsForResource: async function(fileName, sharingIndicatorExpectedToBeVisible) {
-      const resourceRowXpath = this.getFileRowSelectorByFileName(fileName)
-      const shareIndicatorsXpath =
-        resourceRowXpath + this.elements.shareIndicatorsInFileRow.selector
+      let resourceRowXpath = this.getFileRowSelectorByFileName(fileName)
+      let shareIndicatorsXpath = resourceRowXpath + this.elements.shareIndicatorsInFileRow.selector
       const indicators = []
       await this.waitForFileVisible(fileName)
       const {
         waitForNegativeConditionTimeout,
         waitForConditionTimeout,
         waitForConditionPollInterval
       } = client.globals
-      await this.waitForElementVisible({
-        selector: shareIndicatorsXpath,
-        locateStrategy: this.elements.shareIndicatorsInFileRow.locateStrategy,
-        abortOnFailure: false,
-        timeout: sharingIndicatorExpectedToBeVisible
-          ? waitForConditionTimeout
-          : waitForNegativeConditionTimeout,
-        pollInterval: waitForConditionPollInterval
-      })
+      try {
+        await this.waitForElementVisible({
+          selector: shareIndicatorsXpath,
+          locateStrategy: this.elements.shareIndicatorsInFileRow.locateStrategy,
+          abortOnFailure: false,
+          timeout: sharingIndicatorExpectedToBeVisible
+            ? waitForConditionTimeout
+            : waitForNegativeConditionTimeout,
+          pollInterval: waitForConditionPollInterval
+        })
+      } catch (error) {
+        console.log(error)
+        console.log('\n Retrying with new reference \n')
```

Comment:
leftovers?

Reply:
these console logs may be helpful for debugging.

Reply:
maybe the error does not need to be logged but IMO the retrying info is good.